scripts/cat/pelts.py

```diff
@@ -641,12 +642,12 @@ def init_sprite(self):
         self.cat_sprites['senior adult'] = self.cat_sprites['adult']
 
     def init_scars(self, age):
-        if age == "newborn":
+        if age.is_newborn():
             return
 
-        if age in ['kitten', 'adolescent']:
+        if age.is_underage():  # kitten or adolescent now as newborn already excluded
```

Comment:
If the comment is required to aid in understanding the code, I'd prefer it if you renamed the method to `is_kitten_or_adolescent()` or simply `is_kitten() or is_adolescent()`

Reply:
No, it's actually because the check contains newborn, kitten and adolescent. Since newborn is already excluded, this function is OK to use and still means what you think it means.

Reply:
Ah, I skimmed through the PR and didn't see the previous check. With the given context, I don't think the comment is necessary as the code is pretty clear, no?

Reply:
Good point, I'll remove it.